app/src/i18n/locales/de/data.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
This file (I think it's from the german language PR made by the outside contributor?) should be updated to reflect the translation feedback I left on that PR.

Reply:
It's mostly correct though so it's a good start.